### PR TITLE
Kaminari

### PR DESCRIPTION
**Pull request by IgnacioColuccio**
### Summary
- Added AJAX pagination using Kaminari

### Trello card
https://trello.com/c/z1BzVDGh/25-crear-lista-de-articulos-usando-api-rest

### Test plan
1. Go to the articles listing
2. Create articles until there's more than 5
3. A paginator should appear
4. Navigate through the pages using the paginator
5. Its content should get updated without reloading the page

### CODE REVIEW

**Comment by IgnacioColuccio** [comment]
One commit seems to include AJAX e-mail code, I'll fix it when the other pull requests are merged

**Comment by Dinuuu** [comment]
Try to do it before I start correcting it, because there is a lot of code outdated or repeated that I don't have to correct.

**Comment by IgnacioColuccio** [comment]
Sorry, didn't mean to assign it before fixing it. Should be fixed now

**Comment by Dinuuu** [comment]
This PR needs to be rebased

**Comment by IgnacioColuccio** [comment]
Why?

**Comment by IgnacioColuccio** [comment]
Should be OK now, do you see anything wrong with it?

File: .rubocop.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Documentation:
    Enabled: false
 LineLength:
-   Max: 99
\ No newline at end of file
+   Max: 999
```
Comment (Dinuuu) [inline .rubocop.yml]: Got you ! Why did you change this ? :P

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  api_version(module: 'V1', path: { value: 'v1' }, defaults: { format: :json }) do
-    resources :articles, only: [:update]
+  api_version(:module => "V1", path: {value: "v1"}) do
```
Comment (Dinuuu) [inline config/routes.rb]: wasn't this line ok BEFORE you edit it ?

Comment (IgnacioColuccio) [inline config/routes.rb]: Yep! Merge stuff

File: .rubocop.yml (hunk 1 above)
Comment (IgnacioColuccio) [inline .rubocop.yml]: Because there's a line rubocop complains about. I couldn't really shorten it and you had told me to disable that restriciton anyway.

**Comment by IgnacioColuccio** [comment]
Fixed those things

File: app/controllers/v1/articles_controller.rb [hunk 3]
```diff
@@ -10,10 +10,20 @@ def update
       end
     end
 
+    def index
+      @articles = Article.page params[:page]
+      view = render_to_string 'articles/_articles', articles: @articles, layout: false
```
Comment (Dinuuu) [inline app/controllers/v1/articles_controller.rb]: you are rendering the entire view in each request? the advantage of ajax requests is that you just render what changes , here you are rendering EVERYTHING every time

**Comment by Dinuuu** [comment]
:+1:
